apps/docs/src/guide/errors/error-codes.md

```diff
@@ -0,0 +1,71 @@
+# Error Codes
+
+Here is a list of the expected error codes the SDK can throw. These error codes are used to help understand the error that has been thrown with potential remedies.
+
+Error Codes | Description
+--- | ---
+ABI_MAIN_METHOD_MISSING | When your ABI does not have a main method. This can be resolved by adding a main method to your ABI.
+ABI_TYPES_AND_VALUES_MISMATCH | When the arguments supplied to the function do not match the minimum required input length. Check that the arguments supplied to the function, match the required type.
+ACCOUNT_REQUIRED | When the account has not been set for the contract.
+CONVERTING_FAILED | When the `bn` function failed to convert a value supplied.
+DUPLICATED_POLICY | When there are policies with the same type.
+ERROR_BUILDING_BLOCK_EXPLORER_URL | When more than one of the following is options is passed: `path`, `address`, `txId`, `blockNumber`.
+FRAGMENT_NOT_FOUND | When the function fragment is not found in the ABI.
+FUNCTION_NOT_FOUND | When the function with the given name, signature or selector is not found in the ABI.
```

Comment:
Do we need some extra context here around functions and fragments?

Reply:
I personally would favour depreciating `FRAGMENT_NOT_FOUND`, as the `FUNCTION_NOT_FOUND` (from what I can see) is redundant. 

Call to `getFunction` (will throw when not found)
https://github.com/FuelLabs/fuels-ts/blob/19184709f1a0facef9d5d335dcd14e796bb4f259/packages/abi-coder/src/Interface.ts#L62

`fragment` will always be defined after that point
https://github.com/FuelLabs/fuels-ts/blob/19184709f1a0facef9d5d335dcd14e796bb4f259/packages/abi-coder/src/Interface.ts

<img width="893" alt="image" src="https://github.com/FuelLabs/fuels-ts/assets/16990131/e1dfbf86-cc5f-4c13-a60a-e9e096172bc9">

Reply:
I'm happy to deprecate if it's redundant, but maybe do that outside of this PR?

Reply:
That was the plan 👍🏼